package.json

```
@@ -26,7 +32,12 @@
     "last 2 versions"
   ],
   "scripts": {
-    "dev": "gulp",
-    "build": "gulp generate-assets"
+    "dev": "npx webpack watch --config webpack.config.js --mode development",
```

Would it be okay to remove `npx` here? I'm not sure what it does.. but when I tested in Docker it exits immediately if `npx` is included... I don't think it's entirely working and I'm out of time to test, but this way it seems to run in Docker at least...

```suggestion
    "dev": "webpack watch --config webpack.config.js --mode development",
```